.origintrail_noderc.image

```diff
@@ -0,0 +1,38 @@
+{  
+   "node_wallet":"0xb664cf668534fde04cf43173e2187b7a9196bfc3",
+   "node_private_key":"0dbaee2066aacd16d43a9e23649f232913bca244369463320610ffe6ffb0d69d",
+   "management_wallet":"0x64b592e8e9af51eb0dba5d4c18b817c01e8e75a8",
+   "database":{  
+      "password":"root",
+      "provider":"arangodb",
+      "username":"root",
+      "port":8529,
+      "database":"origintrail-development",
+      "host":"localhost",
+      "max_path_length":1000
+   },
+   "traverse_nat_enabled":false,
+   "deposit_on_demand":true,
+   "initial_deposit_amount": "10000000000000000000000",
+   "local_network_only":true,
+   "blockchain":{  
+      "blockchain_title":"Ethereum",
+      "network_id":"rinkeby",
+      "gas_limit":"2000000",
+      "gas_price":"20000000000",
+      "hub_contract_address":"0x0987197628Bb06133B6FA2409eb4cF9FCaFe8d3a",
```

Comment:
We cannot rely the hub address is going to be unchanged. One change in the contracts the whole test will fail.

Reply:
added logic to get hub address from truffle-migrate.log